src/test/java/services/tests/GuestAccountValidateTest.java

```diff
@@ -0,0 +1,127 @@
+package services.tests;
+
+import com.training.base.BaseTest;
+import com.training.services.ga.GAValidationResponse;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+
+public class GuestAccountValidateTest extends BaseTest {
+    @Test
+    public void testGuestAccountValidation() {
+        RestAssured.baseURI = map.get("base_url");
+
+        GAValidationResponse gaValidationResponse =
+                         given()
+                        .header(map.get("AppKeyHeader"), map.get("AppKeyValue"))
+                        .header(map.get("ContentTypeHeader"), map.get("ContentTypeValue"))
+                        .body("{\"email\": \"testPranav@api.com\"}")
+                        .when()
+                        .post("/validation")
+                        .then()
+                        .extract()
+                        .response()
+                        .as(GAValidationResponse.class);
```

Comment:
This looks like duplicate code which is used in all test methods.
Lets create a class - RestEngine, and create a reusable method in that - getResponsePost() - which accepts the headers, body and url. 
And then lets use that resuable method in all test classes